mvpa2/measures/nnsearchlight.py

```diff
@@ -44,7 +44,15 @@ def __init__(self, knn, generator, qe, **kwargs):
           Used to fetch space and dfx settings. TODO
         """
         # verify that desired features are supported
-        if knn.dfx != squared_euclidean_distance:
+        if knn.dfx == squared_euclidean_distance:
+            self._distance = 'euclidean'
+        elif knn.dfx == one_minus_correlation:
+            self._distance = 'correlation'
```

Comment:
and coveralls claims that this code is not tested, not sure how that could have happened